src/zui/ZUIEditor/extensions/LinkExtension.tsx

```diff
@@ -65,6 +66,22 @@ class LinkExtension extends MarkExtension<LinkOptions> {
     };
   }
 
+  /* eslint-disable @typescript-eslint/ban-ts-comment */
+  //@ts-ignore
+  @command()
+  insertEmptyLink(): CommandFunction {
+    return ({ dispatch, state, tr }) => {
+      const mark = this.type.create();
+      const node = this.type.schema.text(String.fromCharCode(160), [mark]);
+      const pos = state.selection.$head.pos;
+      tr = tr.insert(pos, node);
```

Comment:
Someone should check if `tr == tr.insert(pos, node)`, i.e. if the `tr` object changes or is immutable and gets copied.